wp/plugin/lib/parameters.ml

```diff
@@ -35,6 +36,8 @@ type t = {
   show : string list
 }
 
+(* TODO add in validate pointer_reg_list *)
```

Comment:
Are you planning on adding this function in this PR or later?

Reply:
Oops! Since this works on both single and comparative analysis, it's probably not necessary? I deleted the comment.